Makefile

```diff
@@ -203,3 +203,13 @@ _push_tests_cassandra_client_image:
 	cd app_clients/cassandra-client/buildscripts && ./push
 
 cassandra-client: deps _build_tests_cassandra_client_image _push_tests_cassandra_client_image 
+
+_build_tests_pod_delete_image:
+	@echo "INFO: Building container image for performing pod delete chaos"
+	cd app_clients/pod-delete && docker build -t litmuschaos/pod-delete-chaos .
```

Comment:
do we need to add this in app_client or something else? as app-client is more application specific

Reply:
It is also specific to pod delete experiments that's why added here.

Reply:
Yes, It is related to pod-delete not any kind of specific applications like others which are already present there i.e, Cassandra, MySQL, Kafka, etc.